layout/app/catalog/component.js

```diff
@@ -1,42 +1,90 @@
-import React, { PureComponent } from 'react';
-import PropTypes from 'prop-types';
+import React, { Component } from 'react';
```

Comment:
Use always `PureComponent` unless you need to change the `shouldComponentUpdate` behaviour.